lib/render/threejs/three-texture-loader-browser.ts

```diff
@@ -84,14 +84,18 @@ async function load(mimeType: string, objectUrl: string): Promise<ThreeTexture>
 	if (mimeType === 'application/octet-stream') {
 		return await loadHdrTexture(objectUrl);
```

Comment:
not sure, but this wait could be removed as well, right?

Reply:
Yes. Actually I think I've changed the way textures are loaded a few weeks back but didn't adapt all the method signatures. Might be possible to make more methods sync.

Reply:
removed it, think we can improve that later